platform-sdk/swirlds-common/src/main/java/com/swirlds/common/formatting/UnitFormatter.java

```diff
@@ -243,4 +243,9 @@ public void render(final StringBuilder sb) {
             }
         }
     }
+
+    @Override
+    public String toString() {
+        return render();
```

Comment:
That is a bad idea from my point of view. Looks like the render() method can throw exceptions and really does a lot of work. Both should be avoided in `toString`

Reply:
removed